clients/src/main/java/org/apache/kafka/clients/admin/OffsetSpec.java

```diff
@@ -70,4 +71,8 @@ public static OffsetSpec maxTimestamp() {
         return new MaxTimestampSpec();
     }
 
+    public static OffsetSpec earliestLocalTimestamp() {
```

Comment:
Can we add some documentation for this method?

Reply:
Will do! As to your overall question - yeah, I have no problem with adding -5 to this pull request as well. I left this pull request on the backburner until I got the KIP accepted, this is why I haven't touched the code. If you are interested in this feature will you be willing to go over the discussion + voting thread (which I will open beginning of next week) in support?